packages/sui-mono/bin/sui-mono-release.js

```diff
@@ -18,26 +17,34 @@ const RELEASE_CODES = {
   3: 'major'
 }
 
-const releasesByPackages = (pkg) => {
```

Comment:
I've changed how this function works because it made not much sense and would make the mono-package check a bit harder/duplicated.

Before we where requesting for each scope a check and retrieving only the part of the given scope
Now we just perform a check that will return all the scopes (only 1 in the case of monopackage) and work with that information.

It should make the release a bit faster but I don't think it will be noticed by the users